uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -37,145 +50,170 @@ <h4 class="modal-title">New Document</h4>
+              <a class="helpicon" title="Remind Me info" href ng-click="remindMeInfo()"></a>
+            </div>
+          </div>
+        </uib-tab>
+        <uib-tab heading="Assignments">
+          <div class="row double-spacing">
+            <!-- Due Date -->
+            <br>
+            <div class="col-xs-10">
+              <div class="form-group">
+                <label for="{{prefix}}document-due" class="col-xs-3 text-nowrap control-label required-field-indicator">Due Date:</label>
+                <div class="col-xs-9 input-group input-group-unstyled" ng-class="{'has-error': documentForm.dueDate.$invalid}">
+                  <input name="dueDate" type="text" id="{{prefix}}document-due" class="form-control" name="documentDue" ng-model="document.activity_date_time" placeholder="dd/mm/yyyy" uib-datepicker-popup ng-required="true" is-open="dpOpened.due" ng-click="dpOpen($event, 'due')">
+                  <span class="input-group-addon fa fa-calendar"></span>
+                </div>
               </div>
-            </ui-select-choices>
-          </ui-select>
-        </div>
-      </div>
-      <!-- Show More -->
-      <div class="row">
-        <div class="col-xs-12 show-more">
-          <span ng-click="expanded = !expanded;"><i class="fa" ng-class="{'fa-chevron-down': expanded, 'fa-chevron-right': !expanded}"></i> Show more</span>
-        </div>
-      </div>
-      <div class="row" ng-show="expanded">
-        <label class="col-xs-12 col-sm-7" ng-hide="statusFieldVisible()">
-          <span class="form-control-static">
-            <a href class="btn btn-indent required-field-indicator" ng-click="showStatusField()">Set Status</a>
-          </span>
-        </label>
-        <div class="col-xs-12 col-sm-7" ng-show="statusFieldVisible()">
-          <ui-select prevent-animations theme="civihr-ui-select" name="documentStatus" allow-clear class="required-field-indicator" ng-model="document.status_id" ng-required="true">
-            <ui-select-match prevent-animations allow-clear class="ui-select-match" placeholder="Set Status">
-              {{$select.selected.value}}
-            </ui-select-match>
-            <ui-select-choices prevent-animations class="ui-select-choices" repeat="status.key as status in cache.documentStatus.arr | filter: $select.search">
-              <div ng-bind="status.value"></div>
-            </ui-select-choices>
-          </ui-select>
-        </div>
-        <label class="col-xs-12 control-label">Details</label>
-        <div class="col-xs-12">
-          <div text-angular ta-toolbar="[['bold','italics','underline','strikeThrough','ul','ol','undo','redo','clear']]" ng-model="document.details">
+            </div>
           </div>
-        </div>
-      </div>
+          <!-- Assignee & Assignment -->
+          <div class="row">
+            <!-- Assignee -->
+            <div class="col-xs-12 col-sm-7" ng-show="!document.assignee_contact_id[0] && !showAssigneeField">
```

Comment:
why index 0 for `document.assignee_contact_id[0]` , can the length of array be used?